debug: false
emergency_break_horizon: 0.5
force_GBTRACK: false
ftg_active: false
ftg_speed_mps: 0.1
ftg_timer_sec: 3.0
gb_ego_width_m: 0.4
gb_horizon_m: 15.0
lateral_width_gb_m: 0.3
lateral_width_ot_m: 0.3
max_lidar_dist: 9.0
max_speed: 6.0
n_loc_wpnts: 80
overtaking_horizon_m: 6.9
overtaking_ttl_sec: 3.0
pred_splini_ttl: 2.0
range_offset: 180
rate: 80
safety_radius: 40
splini_hyst_timer_sec: 0.7
splini_ttl: 2.0
track_width: 2.6
use_force_trailing: false
volt_threshold: 11
